rescript

```diff
@@ -20,23 +20,24 @@ var LAST_FIRED_EVENT = 0;
 /**
  * @type {[string,string][]}
  */
-var reasons_to_rebuild = [["proj", "started"]];
+var reasonsToRebuild = [["proj", "started"]];
 
 var LAST_SUCCESS_BUILD_STAMP = 0;
 var cwd = process.cwd();
 var lockFileName = path.join(cwd, ".bsb.lock");
 process.env.BSB_PROJECT_ROOT = cwd;
-// console.log('BSB_PROJECT_ROOT:', process.env.BSB_PROJECT_ROOT)
+
+var error_is_tty = process.stderr.isTTY;
+var std_is_tty = process.stdout.isTTY;
 
 // If the project uses gentype and uses custom file extension
 // via generatedFileExtension, ignore them in watch mode
 var bsConfigFile = path.join(cwd, bsconfig);
-var genTypeFileExtension = undefined;
-
+var genTypeFileExtension = ".gen.tsx";
```

Comment:
Prefilled the default value, so we do one check less in the `checkIsRebuildReason` function (previously `valid_event`)